gradle/build-scan.gradle

```diff
@@ -0,0 +1,43 @@
+import nebula.plugin.info.scm.ScmInfoExtension
+
+buildScan {
+    def jenkinsUrl = System.getenv('JENKINS_URL') ? new URL(System.getenv('JENKINS_URL')) : null
+
+    // Accept Gradle ToS when project property org.elasticsearch.acceptScanTOS=true or this is an Elastic CI build
+    if (jenkinsUrl?.host?.endsWith('elastic.co') || Boolean.valueOf(project.findProperty('org.elasticsearch.acceptScanTOS') ?: "false")) {
```

Comment:
we could just ditch the property, I don't see a reason to opt out of CI

Reply:
I use this property to opt-in locally.